node/network/statement-distribution/src/lib.rs

```diff
@@ -1743,7 +1743,7 @@ async fn handle_network_update<Context, R>(
 			// handled by `ActiveLeavesUpdate`
 		},
 		NetworkBridgeEvent::UpdatedAuthorityIds(peer, authority_ids) => {
-			gum::trace!(target: LOG_TARGET, ?peer, "Updated authority ids");
+			gum::trace!(target: LOG_TARGET, ?peer, ?authority_ids, "Updated authority ids");
 			authority_ids.into_iter().for_each(|a| {
```

Comment:
There should be some logic for removing/updating previously present authority-ids, no?

Statement-distribution keeps authority IDs in the peer struct as well, so those should be updated and then used to determine the outdated ones as well.

Reply:
Yes, I have added the removing logic for the outdated authority-ids. Thanks for pointing out that I need to update the authority-ids inside the peer data struct.